javascript/ql/lib/semmle/javascript/frameworks/SQL.qll

```diff
@@ -491,6 +524,31 @@ private module Sequelize {
     }
   }
 
+  /** Gets an import of the `sequelize` module or one that re-exports it. */
```

Comment:
Now that you've converted the `Spanner` model to use MaD, could you do the same with your `sequelize` model?

Reply:
the second time around, it was much easier :)